salt/states/linux_acl.py

```diff
@@ -50,6 +50,22 @@
            - damian
            - homer
          - perms: rwx
+
+.. warning::
+
+    The effective permissions of Linux file access control lists are governed
+    by the effective rights mask bitwise-anded with the `perms` set by this
```

Comment:
**Suggestion (phrasing, non-blocking):** I searched for some articles about ACL masking to get a better sense of what you're trying to communicate here. I don't see the term "bitwise-anded" used much and I think it would be best to avoid technical jargon if it isn't widely used or generally understood. Can you remove that term or rephase it to use something that communicates in plainer English?

Reply:
I've rewritten a bit to be hopefully-clearer.